openvpn/middlewares/server/auth/middleware.go

```diff
@@ -170,7 +173,7 @@ func (m *middleware) authenticateClient() (consumed bool, err error) {
 
 	log.Info("authenticating user: ", m.lastUsername, " clientID: ", m.clientID, " keyID: ", m.keyID)
 
-	authenticated, err := m.authenticator(m.lastUsername, m.lastPassword)
+	authenticated, err := m.checkCredentials(m.lastUsername, m.lastPassword)
 	if err != nil {
 		return false, err
```

Comment:
If some error occurs in authenticator, we leave openvpn client waiting - maybe we should reply with `client-deny`, just as when password is incorrect? We can specify custom message in `client-deny` message, i.e. `internal server error occured`.

Reply:
Functionality added. Deny client login on credentials checking error